text/0026-finality-signature-gossiping.org

```diff
@@ -13,8 +13,9 @@
 CEP PR: [[https://github.com/casperlabs/ceps/pull/0026][casperlabs/ceps#0026]]
 
 Nodes joining the network will use /finality signatures/ to securely
-synchronize with the validators. As a node synchronizes, it will get
-signatures of block hashes via the gossip network until it has > 33%
+synchronize with the validators.  These nodes were identified in As a
```

Comment:
```suggestion
synchronize with the validators.  As a
```
Probably a typo, or missing sentence?

Reply:
Typo yeah.

Reply:
Fixed in https://github.com/CasperLabs/ceps/pull/26/commits/bf15965deb13ab20a1ebf2b8489e3796232354db